### PR TITLE
restrict homes to 10.1.1.1-10.1.1.26, clean up quotes

### PR DESCRIPTION
**Pull request by jordane**

### CODE REVIEW

File: recipes/homes.rb [hunk 1]
```diff
@@ -17,10 +17,10 @@
 # limitations under the License.
 #
 
-include_recipe "osl-nfs::default"
+include_recipe 'osl-nfs::default'
 
-nfs_export "/data/homes/" do
-  network '10.1.0.0/23'
+nfs_export '/data/homes/' do
+  network (1..26).map { |octet| '10.1.1' + octet.to_s + '/32' }
```
Comment (armiller) [inline recipes/homes.rb]: I think you need a `10.1.1.` here otherwise it will be `10.1.11`.

Comment (jordane) [inline recipes/homes.rb]: Yep.

**Comment by armiller** [comment]
:+1:

**Comment by armiller** [comment]
:+1:

**Pull request merged**